distribution/docs/src/main/resources/_contents/_endpoints/metrics-endpoint-contents.adoc

```diff
@@ -0,0 +1,221 @@
+_This parameter cannot be used with the `dateOffset` parameter._
+|`endDate=2013-04-01T11:00:00-07:00`
+
+|dateOffset
+|Specifies an offset, backwards from the current time, to search on the modified time field for entries. Defined in seconds and must be a positive Integer. +
+_This parameter cannot be used with the `startDate` or `endDate` parameters._
+|`dateOffset=1800`
+
+|yAxisLabel
+|(optional) the label to apply to the graph's y-axis. Will default to the metric's name, e.g., Catalog Queries. +
+_This parameter is only applicable for the metric's graph display format.
+|Catalog Query Count
+
+|title
+|(optional) the title to be applied to the graph.
+
+Will default to the metric's name plus the time range used for the graph.
+
+_This parameter is only applicable for the metric's graph display format._
+|Catalog Query Count for the last 15 minutes
+
+|===
+
+====== Metrics Data Supported Formats
+
+The metric's historical data can be displayed in several formats, including PNG , a CSV file, an Excel .xls file, a PowerPoint .ppt file, an XML file, and a JSON file.
+The PNG, CSV, and XLS formats are accessed via hyperlinks provided in the Metrics tab web page.
+The PPT, XML, and JSON formats are accessed by specifying the format in the custom URL, e.g., \http://<${branding}_HOST>:<${branding}_PORT>/services/internal/metrics/catalogQueries.json?dateOffset=1800.
+
+The table below describes each of the supported formats, how to access them, and an example where applicable. (NOTE: all example URLs begin with \${secure_url} which is omitted in the table for brevity.)
+
+.Metrics Formats
+[cols="1,2,1,5a" options="header"]
+|===
+
+|Display Format
+|Description
+|How To Access
+|Example URL
+
+|PNG
+|Displays the metric's data as a PNG-formatted graph, where the x-axis is time and the y-axis is the metric's sampled data values.
+
+|Via hyperlink on the Metrics tab or directly via custom URL.
+|Accessing Catalog Queries metric data for last 8 hours (8 * 60 * 60 = 28800 seconds):
+
+`/services/internal/metrics/catalogQueries.png?dateOffset=28800&yAxisLabel=my%20label&title=my%20graph%20title`
+
+Accessing Catalog Queries metric data between 6:00 am on March 10, 2013, and 10:00 am on April 2, 2013:
+
+``/services/internal/metrics/catalogQueries.png?startDate=2013-03-10T06:00:00-07:00&endDate=2013-04-02T10:00:00-07:00&yAxisLabel=my%20label&title=my%20graph%20title`
+
+_Note that the `yAxisLabel` and `title` parameters are optional_.
+
+|CSV
+|Displays the metric's data as a Comma-Separated Value (CSV) file, which can be auto-displayed in Excel based on browser settings.
+
+The generated CSV file will consist of two columns of data: Timestamp and Value, where the first row are the column headers and the remaining rows are the metric's sampled data over the specified time range.
```

Comment:
Suggestion: _where the first row **contains**_ and _the remaining rows **contain**_